h2/src/test/org/h2/test/scripts/datatypes/enum.sql

```diff
@@ -107,22 +107,36 @@ alter table card alter column suit enum('hearts', 'clubs', 'spades', 'diamonds',
 drop table card;
 > ok
 
+CREATE TYPE my_number AS NUMBER;
+> ok
+
+alter type my_number ADD VALUE 'diamonds';
+> exception WRONG_OBJECT_TYPE
+
+drop type my_number;
+> ok
+
 --- ENUM as custom user data type
 
-create type CARD_SUIT as enum('hearts', 'clubs', 'spades', 'diamonds');
+create type CARD_SUIT as enum('hearts', 'clubs', 'spades');
 > ok
 
 create table card (rank int, suit CARD_SUIT);
 > ok
 
-insert into card (rank, suit) values (0, 'clubs'), (3, 'hearts');
+@reconnect
```

Comment:
We need to test domain modified with `ADD VALUE` and column with this domain after reconnection, move this line below.

Reply:
Fixed